openassessment/assessment/models/ai.py

```diff
@@ -1,874 +1 @@
-
-    def assign_most_recent_classifier_set(self):
-        """
-        Find the most recent classifier set and assign it to this workflow.
-
-        Returns:
-            (bool) indicates whether or not classifiers were able to be assigned to the AIGradingWorkflow
-
-        Raises:
-            DatabaseError
-        """
-        classifier_set = AIClassifierSet.most_recent_classifier_set(
-            self.rubric, self.algorithm_id, self.course_id, self.item_id
-        )
-        if classifier_set is not None:
-            self.classifier_set = classifier_set
-            self.save()
-        return classifier_set is not None
-
-    @classmethod
-    @transaction.atomic
-    def start_workflow(cls, submission_uuid, rubric_dict, algorithm_id):
-        """
-        Start a grading workflow.
-
-        Args:
-            submission_uuid (str): The UUID of the submission to grade.
-            rubric_dict (dict): The serialized rubric model.
-            algorithm_id (unicode): The ID of the algorithm to use for grading.
-
-        Returns:
-            AIGradingWorkflow
-
-        Raises:
-            SubmissionNotFoundError
-            SubmissionRequestError
-            SubmissionInternalError
-            InvalidRubric
-            DatabaseError
-
-        """
-        # Retrieve info about the submission
-        submission = sub_api.get_submission_and_student(submission_uuid)
-
-        # Get or create the rubric
-        from openassessment.assessment.serializers import rubric_from_dict
-        rubric = rubric_from_dict(rubric_dict)
-
-        # Create the workflow
-        workflow = cls.objects.create(
-            submission_uuid=submission_uuid,
-            essay_text=essay_text_from_submission(submission),
-            algorithm_id=algorithm_id,
-            student_id=submission['student_item']['student_id'],
-            item_id=submission['student_item']['item_id'],
-            course_id=submission['student_item']['course_id'],
-            rubric=rubric
-        )
-
-        # Retrieve and assign classifier set candidates
-        workflow.assign_most_recent_classifier_set()
-
-        workflow._log_start_workflow()
-
-        return workflow
-
-    @transaction.atomic
-    def complete(self, criterion_scores):
-        """
-        Create an assessment with scores from the AI classifiers
-        and mark the workflow complete.
-
-        Args:
-            criterion_scores (dict): Dictionary mapping criteria names to integer scores.
-
-        Raises:
-            InvalidRubricSelection
-            DatabaseError
-
-        """
-        self.assessment = Assessment.create(
-            self.rubric, self.algorithm_id, self.submission_uuid, AI_ASSESSMENT_TYPE
-        )
-        AssessmentPart.create_from_option_points(self.assessment, criterion_scores)
-        self.mark_complete_and_save()
+# This file is empty, but we cannot delete it b/c historical migration records refer to it.
```

Comment:
I will double check on this one - it could be in a weird intermediary state from my internal debate about dealing with migrations.